memory/ndctl.py

```diff
@@ -551,6 +562,83 @@ def test_region_capabilities(self):
                     region=reg_name, mode='devdax', align=size)
                 self.plib.destroy_namespace(region=reg_name, force=True)
 
+    @avocado.fail_on(pmem.PMemException)
+    def write_infoblock(self, ns_name, align):
+        """
+        Write_infoblock on given namespace
+        """
+        if not self.plib.check_ndctl_subcmd("write-infoblock"):
+            self.cancel("Binary does not support write-infoblock")
```

Comment:
do you want to cancel or fail here, considering we are in `test_X()` already ?

Reply:
`cancel` is more suitable considering most of the distro's ndctl package does not contain `write-infoblock` sub-command yet (fairly newer command). So it is kind of not supported with the distribution's ndctl package, and hence skipping here.